apps/backend/functions/src/routes/tracks.ts

```diff
@@ -14,7 +14,8 @@ export const getTracks = onCall(async (request) => {
                 title: t.data().title,
                 image: t.data().image,
                 description: t.data().description,
-                problems: t.data().problems.map(x => x.id)
+                problems: t.data().problems.map(x => x.id),
```

Comment:
These should be filtered from the backend itself. 
Also added a comment here - https://github.com/code100x/daily-code/issues/13#issuecomment-1921228116

Reply:
Made the change